Unmasked/Unmasked View/View Controllers/UnmaskedViewController.swift

```diff
@@ -0,0 +1,259 @@
+//
+//  UnmaskedViewController.swift
+//  Unmasked
+//
+//  Created by Junaid Rajah on 2021/11/19.
+//
+
+import UIKit
+import UnmaskedEngine
+
+class UnmaskedViewController: UIViewController {
+    
+    private var unmaskedViewModel: UnmaskedViewModel?
+    public var heroGroup: Int = 0
+    @IBOutlet private weak var heroNameLabel: UILabel!
+    @IBOutlet private weak var heroImageView: UIImageView!
+    @IBOutlet private weak var infoScreenTitleLable: UILabel!
+    @IBOutlet private weak var heroInfoView: UIView!
+    
+    @IBOutlet private weak var menuImage: UIImageView!
+    @IBOutlet private weak var menuOuterImage: UIImageView!
+    @IBOutlet private weak var powerButton: UIButton!
+    @IBOutlet private weak var appearanceButton: UIButton!
+    @IBOutlet private weak var biographyButton: UIButton!
+    @IBOutlet private weak var workButton: UIButton!
+    @IBOutlet private weak var connectionButton: UIButton!
+    @IBOutlet private weak var unmaskedMenuView: UIView!
+    
+    @IBOutlet private weak var biographyBorder: UIImageView!
+    @IBOutlet private weak var connectionBorder: UIImageView!
+    @IBOutlet private weak var workBorder: UIImageView!
+    @IBOutlet private weak var appearanceBorder: UIImageView!
+    @IBOutlet private weak var powerBorder: UIImageView!
+    
+    public var hero: SuperheroResponseModel?
```

Comment:
Please send this to your `ViewModel` your ViewController shouldn't know what `SuperheroResponseModel` is.